NEWS.md

```diff
@@ -49,6 +52,7 @@ Standard library changes
 * A no-argument construct to `Ptr{T}` has been added which constructs a null pointer ([#30919])
 * `strip` now accepts a function argument in the same manner as `lstrip` and `rstrip` ([#31211])
 * `mktempdir` now accepts a `prefix` keyword argument to customize the file name ([#31230], [#22922])
+* `nextfloat(::BigFloat)` and `prevfloat(::BigFloat)` now uses the precision of the argument ([#31310])
```

Comment:
nitpick, if you happen to re-push before merge: "uses" -> "use"

Reply:
```suggestion
* `nextfloat(::BigFloat)` and `prevfloat(::BigFloat)` now return a value with the same
   precision as their argument, which means that (in particular)
   `nextfloat(prevfloat(x)) == x` whereas previously this could result in a completely
   different value with a different precision ([#31310])
```